packages/flutter/lib/src/material/banner.dart

```diff
@@ -18,6 +18,30 @@ import 'theme.dart';
 /// They are persistent and non-modal, allowing the user to either ignore them or
 /// interact with them at any time.
 ///
+/// {@tool dartpad --template=stateless_widget_scaffold}
+///
+/// ```dart
+/// Widget build(BuildContext context) {
+///   return MaterialBanner(
+///     padding: const EdgeInsets.all(20),
+///     content: Text("Hey, I am an Material Banner"),
```

Comment:
`I am an Material Banner` here is a grammatical mistake.
Here should be `I am a Material Banner`
Thanks.

Reply:
@Abhishek01039 good catch - do you want to fix this in a separate PR?

Reply:
Yes.